### PR TITLE
Add `excluded_services` option for Azure integration (#41)

### LINKED ISSUE #41: Add exclusion ability on azure integration
Hello

Is-it possible to add a param to exclude one or multiples services (like Virtual Machine) from the signalfx discovered services ?

As VM monitor VM with an external solution (like zabbix) we want to disable VM discovery by the signalfx integration.

Regards

HNo

### PR DESCRIPTION
**Pull request by Shr3ps**
close #41

### CODE REVIEW

File: cloud/azure/sfx/integrations-azure.tf [hunk 1]
```diff
@@ -12,7 +12,7 @@ resource "signalfx_azure_integration" "azure_integration" {
   app_id     = var.azure_sp_application_id
   secret_key = var.azure_sp_application_token
 
-  services = coalescelist(var.services, data.signalfx_azure_services.azure_services.services[*].name)
+  services = setsubtract(local.azure_services, var.excluded_services)
```
Comment (xp-1000) [inline cloud/azure/sfx/integrations-azure.tf:15]: I would applied the setsubstract only on the full service list from the datasource, it seems a bit weird but I suppose it should not be a problem while it is few probable a user provide a same service for both `services` and `excluded_services` variables.

File: cloud/azure/variables.tf [hunk 2]
```diff
@@ -38,6 +38,12 @@ variable "services" {
   default     = []
 }
 
+variable "excluded_services" {
+  description = "List of Azure services to not collect metrics for (do not add an include namespace_sync_rule)"
```
Comment (xp-1000) [inline cloud/azure/variables.tf:42]:
```suggestion
  description = "List of Azure services to not collect metrics for (removed from the `services` list)"
```

**Review by xp-1000 — CHANGES_REQUESTED** [review]
thanks,

if you want you can add a note to the readme to inform the user of `additionalServices` feature to collect metrics of not officially supported azure services but without tags sync.

File: cloud/azure/sfx/integrations-azure.tf (hunk 1 above)
Comment (Shr3ps) [reply cloud/azure/sfx/integrations-azure.tf:15]: Hum, you're too optimistic with user usage of modules 😹 

This way we're sure excluded elements are always excluded :)

**Review by xp-1000 — APPROVED** [review]
looks good

**Pull request merged**